js/client/modules/@arangodb/testsuites/importing.js

```diff
@@ -300,6 +300,24 @@ const impTodos = [{
   coll: 'UnitTestsImportEdgeGz',
   type: 'json',
   create: 'false'
+}, {
+  id: 'edgeRewriteOn',
+  data: tu.makePathUnix(fs.join(testPaths.importing[1], 'import-edges.json')),
+  coll: 'UnitTestsImportEdgeRewriteCollectionOn',
+  type: 'json',
+  create: 'false',
+  ["from-collection-prefix"]: 'UnitTestsImportJson1',
+  ["to-collection-prefix"]: 'UnitTestsImportJson2',
+  ["overwrite-collection-prefix"]: true
```

Comment:
I understand that these attribute names have to put into quotes, because they contain punctuation.
I don't understand why these attributes names have to put inside arrays though. Is there a reason for this?
Same below.

Reply:
Already fixed, JSLint aggress with you, it is NOT necessary, my memory was wrong :+1: